packages/studio/src/store/useStudioStore.ts

```diff
@@ -56,6 +56,10 @@ const useStudioStore = create<StudioStore>()(
 );
 
 void useStudioStore.getState().actions.refreshActivePageEntities();
+void (
+  useStudioStore.getState().studioConfig.isPagesJSRepo &&
```

Comment:
Should we guard refreshActivePageEntities with isPagesJSRepo?

Reply:
It's not necessary because `refreshActivePageEntities` looks for the `pagesJS.entityFiles` part of the active page's state. And since that doesn't exist for pages in a non-PagesJS repo, `activePageEntities` is set to undefined, which is the correct behavior